src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -113,6 +114,11 @@ public void addClient(Client client) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void addProcedure(Procedure procedure) {
```

Comment:
I feel like this should exist as a separate class since we have a separate AddProcCommand class

Reply:
Hi JB, Oh yes I agree. This method exists as a separate class, but it has to be implemented because in this ModelStub because a ModelStub implements from a Model, which has the following functions as well.